README.md

```diff
@@ -163,7 +163,7 @@ $ # here we use the token created above
 $ PARENTURL=https://192.168.10.10 REGTOKEN=07401b.f395accd246ae52d envsubst < ./deploy/templates/clusternet_agent_secret.yaml | kubectl apply -f -
 ```
 
-The `PARENTURL` above is the apiserver address of the parent cluster that you want to register to.
+The `PARENTURL` above is the apiserver address of the parent cluster that you want to register to. If the apiserver is not listening on the standard https port (:443), please specify the port number in the URL to ensure the agent connects to the right endpoint, for instance, `https://192.168.10.10:6443`.
```

Comment:
And http/https scheme should be included as well.

Reply:
updated, please review again